pom.xml

```diff
@@ -84,7 +84,12 @@ under the License.
   </distributionManagement>
 
   <properties>
-    <javaVersion>8</javaVersion>
+    <javaVersion>11</javaVersion>
+    <maven.compiler.source>${javaVersion}</maven.compiler.source>
```

Comment:
What this is for? Given that
```
error: option -source cannot be used together with --release
error: option -target cannot be used together with --release
Usage: javac <options> <source files>
use --help for a list of possible options
```
and if release is set (and in new line 359 it is) - source/target are not used by compiler-plugin anyway.

Reply:
It's in parent, so here is just redefined (as see in parent what is happening), and yes, it works as release is set... basically, just to be aligned if some tool relies on this

Reply:
See parent here https://github.com/apache/maven-parent/blob/46af4d6a5cf3b49a116e2f52e17c8343e33805e2/pom.xml#L928-L930 (this commit is parent v 36)